src/unittests/conversion.spec.ts

```diff
@@ -285,3 +286,30 @@ g.test('pack2x16float')
       `pack2x16float(${inputs}) returned [${got_str}]. Expected [${expect}]`
     );
   });
+
+g.test('pack2x16snorm')
+  .paramsSimple([
+    // Normals
+    { inputs: [0, 0], result: 0x00000000 },
+    { inputs: [1, 0], result: 0x00007fff },
+    { inputs: [0, 1], result: 0x7fff0000 },
+    { inputs: [1, 1], result: 0x7fff7fff },
+    { inputs: [-1, -1], result: 0x80018001 },
+    { inputs: [0.1, 0.1], result: 0x0ccd0ccd },
+    { inputs: [-0.1, -0.1], result: 0xf333f333 },
+    { inputs: [0.5, 0.5], result: 0x40004000 },
+    { inputs: [-0.5, -0.5], result: 0xc001c001 },
+    { inputs: [0.1, 0.5], result: 0x40000ccd },
+    { inputs: [-0.1, -0.5], result: 0xc001f333 },
```

Comment:
Please can we have some tests for > 1 and < -1?

Reply:
Done